integration-tests/infinispan-common/src/main/java/org/apache/camel/quarkus/component/infinispan/common/InfinispanCommonRoutes.java

```diff
@@ -110,11 +110,12 @@ public void configure() {
                 .setHeader(InfinispanConstants.KEY).constant("the-key")
                 .setHeader(InfinispanConstants.VALUE).body()
                 .toF("infinispan:%s", CACHE_NAME);
-
         from("direct:query")
                 .setHeader(InfinispanConstants.OPERATION).constant(InfinispanOperation.QUERY)
                 .toF("infinispan:%s", CACHE_NAME);
-
+        from("direct:infinispan-query")
+                .setHeader(InfinispanConstants.OPERATION).constant(InfinispanOperation.QUERY)
+                .toF("infinispan-query:%s", CACHE_NAME);
```

Comment:
We'll need to move this out of the common routes and into the `infinispan` module. Otherwise it'll impact the `infinispan-quarkus-client` tests where the `infinispan-query` component bean is not produced (or needed).

Reply:
I'll do it, my bad i didn't try to run the other module.